Brain/src/rising_plugin/guardrails-config/actions/actions.py

```diff
@@ -44,20 +42,20 @@
     FALCON_7B,
 )
 
-"""
-query is json string with below format
-{
-    "query": string,
-    "model": string,
-    "uuid": string,
-    "image_search": bool,
-}
-"""
+from Brain.src.rising_plugin.pinecone_engine import (
+    get_pinecone_index_namespace,
+    init_pinecone,
+)
+
+
+def get_pinecone_index_train_namespace() -> str:
```

Comment:
It's better to move this method to train service.

Reply:
done